### ISSUE TITLE
Add POS tagging 

### ISSUE THREAD

**Issue by jbesomi** [jbesomi/texthero#36]
Under the nlp module, add part-of-speech tagging `hero.pos`. Again here the solution would be to use spaCy. This should not be particularly complex as the code would resemble `named_entities` or `noun_chunks`.

Food for thoughts: having separate functions allows for simple code, but it makes the pipeline quite inefficient as every time an NLP function is called spaCy has to go through all the corpus again. Other ideas and solutions are also welcomed.

Will be implemented only after #65 is solved.

Linked PR: #57

How to contribute: [CONTRIBUTE.md](https://github.com/jbesomi/texthero/blob/master/CONTRIBUTING.md)

**Comment by Rickecr** [comment]
Hi, is this issue open for new contributions or it is already being resolved?

**Comment by jbesomi** [comment]
Hi @Rickecr and welcome to Texthero!

Some days ago @samirghouri started working on this issue (see PR #57) but he left the work unfinished.

As we haven't heard from him since 8 days, I would say you can take over this issue.

I encourage you to first read [CONTRIBUTING.md](https://github.com/jbesomi/texthero/blob/master/CONTRIBUTING.md) and [PURPOSE.md](https://github.com/jbesomi/texthero/blob/master/PURPOSE.md) as well as the review comments in #57.

You can either work on a new PR or edit the one of @samirghouri, probably it will be easier for you to start a new one (less merge conflicts and you will understand better the code).

Thank you for your help, for any question or remarks I'm available! 👍 
regards,

**Comment by henrifroese** [comment]
Implemented and merged in #106
